packages/test/end-to-end-tests/src/test/fluidObjectHandle.spec.ts

```diff
@@ -118,14 +61,18 @@ describe("FluidObjectHandle", () => {
 
         // Verify that the local client's FluidDataObjectRuntime has the correct absolute path.
         const fluidHandleContext11 = firstContainerObject1._runtime.rootRoutingContext;
-        assert.equal(fluidHandleContext11.absolutePath, absolutePath, "The FluidDataObjectRuntime's path is incorrect");
-
-        // Verify that the remote client's FluidDataObjectRuntime has the correct absolute path.
-        const fluidHandleContext12 = secondContainerObject1._runtime.rootRoutingContext;
-        assert.equal(
-            fluidHandleContext12.absolutePath,
-            absolutePath,
-            "The remote FluidDataObjectRuntime's path is incorrect");
+        // back-compat for N-2 <= 0.27, remove when N-2 >= 0.28
+        if (fluidHandleContext11) {
```

Comment:
Why is back-compat necessary here? I don't know if I like doing it this way. If the predicate is false, we basically test nothing but report the test as passing.

Reply:
rootRoutingContext is introduced in 0.28.   I don't like it either, I am going to do a change next to expose the version of test-utils.  but that won't take affect until N-2 == 0.28

Reply:
Could this test just go in its own function that gets passed to generateTest() instead of generateTestWithCompat()?

Reply:
Please see the latest change.  I imported the old version and disable the test if it doesn't meet the requirement.

Reply:
Doesn't this mean the test won't run at all? I thought the goal was for it to run with current-version only, until N-2 >= 0.28 when it will run compat tests too.

I'm also not sure how it will work long-term to reason about minor version only (what about 1.0?)

Reply:
oh.. you are right.  that doesn't work.  
I was trying to continue down the path of putting the full version information on the LocalTestObjectProvider, but it got complicated real quick.  So I reverted this change to go back tot the original if check, which should automatically lite up as we go up the version.  We should clean this up afterward.

I did make one modification to add this.skip() to indicate that the test has been skipped though.

Reply:
It does seem like this introduces a gap where the functionality being tested could fail but we wouldn't notice.

Reply:
Yes. An official way to do version check for interfaces approach maybe the long term solution.  But for now, it should be ok. Do you have any other suggestions?

Reply:
I will merge for now, and continue to iterate on a better solution